tests/functional/index-non-auth.js

```diff
@@ -30,7 +30,7 @@ define([
     'reporter addon link is shown': function() {
       return this.remote
         .get(require.toUrl(url('/')))
-        .findByCssSelector('.wc-Navbar-link').getVisibleText()
+        .findByCssSelector('.js-Navbar-link').getVisibleText()
```

Comment:
Hum yes, my convention is namespace-Component-children--modifier. I think it's good to have a constistent ClassName convention. I don't know what do you want for JS ? js-Navbar-link is equal to the className for the markup wc-Navbar-link